### DISCUSSION TITLE
Should we need a specific case study for this project?

### DISCUSSION THREAD

**Discussion by panoti** [nashtech-garage/nodejs-msa#35] [General]
Hi everyone, hi anh @VuiLenDi Should we need a specific case study for this project? Like bookstore (anh @sonck201 boilerplated), storefront. I think we also need some requirements such as use-cases, bla bla

**Comment by sonck201** [comment]
Yeah, should think about of it, I want to have a specific case study but don't want to be the same with the current one, so bookstore is born, 
But if you have any other ideas that's suitable, just go ahead & propose

**Comment by hungxnguyen118** [comment]
sumary we use the general name of demo project is bookstore

**Comment by panoti** [comment]
Some highlight use cases:
- Guest (Non-authenticated user) can login
- Guest can register
- Member (Authenticated user) can logout
- Member can add/remove wish list
- Member can rent a book
- Member can deposit
- Member can give book back
- Member can do a full payment
- Member can view history of rental
- User (guest and member) can search books
- User can view book detail
- Lessor can login
- Lessor can add/remove book
- Lessor can approve the rental
- Lessor can view the list of rentals

**Reply by sonck201** [reply]
these highlights have been added to 
https://github.com/nashtech-garage/nodejs-msa/pull/45
